.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# How to contribute to MFTF docs
```

Comment:
The "documentation" word is used 8 times in this section.
Rephrase to avoid the unneeded repetition. For example:
_We welcome contributions to the MFTF documentation, which is kept within the /docs/ folder in this repository._
_This page describes the submitting process and serves as a template for a properly written content._
_The contribution workflow is the same as submitting code._
_The content in this repository is also a source for the [MFTF guide topics][]._

Reply:
Done.